try_solver.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
```

Comment:
Do you have something in mind with this file? Do you want to remove it or convert it into an example?

Reply:
it was just to debug, I have removed it